src/language.rs

```diff
@@ -60,8 +60,8 @@ impl std::fmt::Display for Language {
             Language::C => write!(f, "C"),
             Language::Clojure => write!(f, "Clojure"),
             Language::CoffeeScript => write!(f, "CoffeeScript"),
-            Language::Cpp => write!(f, "C++"),
-            Language::Csharp => write!(f, "C#"),
+            Language::Cpp => write!(f, "Cpp"),
+            Language::Csharp => write!(f, "Csharp"),
```

Comment:
Unfortunately, I was hoping for the valid *inputs* to be changed to "C++", "C#", etc. \(preferably case-insensitive\), instead of the *outputs* being changed to "Cpp", "Csharp", etc.

Reply:
This may be a stupid question, but how does `Language::from_str` works?
There is no explicit implementation of the `FromStr`.
Is it done implicitly via `#[strum(serialize_all = "lowercase")]` or `EnumString`.
The answer would help in resolving the issue with the valid inputs

Reply:
`Language::from_str` is implemented from [`#[derive(EnumString)]`](https://docs.rs/strum_macros/0.16.0/strum_macros/derive.EnumString.html). Documentation can be found [here](https://github.com/Peternator7/strum/wiki/Derive-EnumString).

Hope this helps!